src/core/datatypes/phylogenetics/character/DoubletState.cpp

```diff
@@ -79,11 +79,11 @@ std::string DoubletState::getStringValue(void) const
     RbBitSet letters1(4);
     RbBitSet letters2(4);
 
-    for(int i=0;i<state.size();i++)
+    for (int i=0;i<state.size();i++)
     {
         if (state[i])
         {
-            auto& doublet = DOUBLETS[i];
+            const std::string& doublet = DOUBLETS[i];
```

Comment:
Changing code that someone else wrote to suit your style preference seems a bit arbitrary.